source/develop/developer-guide/engine/engine-supported-time-zones.md

```diff
@@ -1,19 +1,27 @@
 ---
 title: Engine supported Time Zones
-authors: mbetak
+authors:
+- mbetak
+- sabusale
 ---
 
 # Engine supported Time Zones
 
-Due to the fact the supported timezone list needs to be shared between the engine frontend, backend and also the engine-config tool the list of supported time zones is concentrated in
+Engine default time-zones can be found in `<ENGINE_DEPLOYMENT>/etc/ovirt-engine/timezones/00-defaults.properties`
 
-<http://gerrit.ovirt.org/gitweb?p=ovirt-engine.git;a=blob;f=backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/TimeZoneType.java>
+####Timezones file format:
 
-There are two basic sets of timezone keys:
+key must be valid timezone from tz database, value must be valid windows timezone
```

Comment:
a valid Windows timezone*
and how about pointing to http://en.wikipedia.org/wiki/Tz_database and http://msdn.microsoft.com/en-us/library/ms912391(v=winembedded.11).aspx as before?